meta/run.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# In this file we specify how each microservice should boot.
+#
+
+runtime: native
+
+config_set: 
+   keyvaluestore:
+      bootcmd: /node keyvaluestore.js
+      env:
+         PORT: 9000
+   db:
+      bootcmd: /node db.js
+      env:
+         PORT: 9001
+         MICRO_KEYVALUESTORE_ENDPOINT: {IP}:9000
```

Comment:
No, this is not meant to be replaced by machine at the moment. User should set it like this:
1) open meta/run.yaml in Notepad
2) manually replace `{IP}` with actual IP address of keyvaluestore
3) only then use `capstan package compose micro`
4) and the `capstan run micro` afterwards